tests/frontend/unit/test.focal-point.js

```diff
@@ -61,7 +61,7 @@ describe('Cl.FocalPoint', function () {
             });
 
             expect(updateLocationValueStub).toHaveBeenCalled();
-            expect(updateLocationValueStub.calls.count()).toBe(1);
+            expect(updateLocationValueStub.calls.count()).toBe(2);
```

Comment:
Not really sure why would this change, since it doesn't seem to be failing on master, but the rest looks good!

Reply:
apparently the testing framework now counts the invocations globally.